### DISCUSSION TITLE
OpenSSL Vulnerabilities?

### DISCUSSION THREAD

**Discussion by dsgoody** [datalust/seq-tickets#1680] [Help] [answered]
Hello,

Seq appears to contain an implementation of OpenSSL within the following files;
- C:\Program Files\Seq\System.Security.Cryptography.OpenSsl.dll
- C:\Program Files\Seq\Client\System.Security.Cryptography.OpenSsl.dll

Can somebody confirm if this is vulnerable to the recently released CVE-2022-3602 and CVE-2022-3786 vulnerabilities affecting OpenSSL 3.x?

Many thanks.

**Comment by nblumhardt** [comment, marked as answer]
Hi @dsgoody - thanks for reaching out.

We're still analyzing the impact of this one, however our initial investigation suggests that the impact is low for Seq servers: Seq itself doesn't use client certificate TLS authentication, and doesn't allow non-administrative users to initiate connections to untrusted servers.

In case a server is configured with e.g. plug-in apps that open up exploitation opportunities, it appears that the issue would more than likely result in a crash rather than RCE.

Nonetheless we believe it will be prudent to release a Seq version with appropriate patches applied as soon as possible; we'll open an issue so that you can subscribe to updates.

Best regards,
Nick

**Reply by nblumhardt** [reply]
https://github.com/datalust/seq-tickets/issues/1682

**Reply by dsgoody** [reply]
Hi Nick,

Thanks for the helpful response, it sounds very reassuring.

I do agree, it's better to patch and be sure the vulnerability doesn't exist than risk it being an issue.

Dave
